### DISCUSSION TITLE
Node Repl Not Working on Serial Console

### DISCUSSION THREAD

**Discussion by cemalgnlts** [copy/v86#874] [Q&A]
Hi,

I am trying to start `Nodejs` in `repl` mode. I try it first in GUI (`/dev/console`) and it works successfully.
I then try to run it in the console (`/dev/ttyS0`) and it throws a strange error.

![Screenshot 2023-06-06 16 43 52](https://github.com/copy/v86/assets/45357531/b206d299-1c6a-4947-a642-7b84fb37ef3a)

The ISO file can be obtained from here:  [linux.iso](https://app.circleci.com/pipelines/github/cemalgnlts/now-linux-build/95/workflows/80c50180-95ad-487d-bc20-f0c257778d41/jobs/103/artifacts)

**Comment by copy** [comment]
I can reproduce this, so converted into an issue: https://github.com/copy/v86/issues/875
